### DISCUSSION TITLE
how to build onnxruntime with latest version of onnx

### DISCUSSION THREAD

**Discussion by ZchiPitt** [microsoft/onnxruntime#20629] [Other Q&A]
### Describe the issue

we have a need to run bf16 model in ort, while onnx just added support for bf16 for many ops in Opset22 (https://github.com/onnx/onnx/commit/4e7289d05a08453b3f20efeb774d4590b5428903)

What is the best way to build the ORT with the latest onnx version? We tried by following https://github.com/microsoft/onnxruntime/blob/main/docs/How_To_Update_ONNX_Dev_Notes.md.

However we are still hitting the errors:
 INVALID_GRAPH : Load model from down.onnx failed:This is an invalid model. In Node, ("/block/resnets.0/norm1/Reshape", Reshape, "", -1) : ("sample_in": tensor(bfloat16),"/block/resnets.0/norm1/Constant_output_0": tensor(int64),) -> ("/block/resnets.0/norm1/Reshape_output_0": tensor(bfloat16),) , Error No Op registered for Reshape with domain_version of 22

### To reproduce

NA

### Platform

Windows

### OS Version

11

### ONNX Runtime Installation

Built from Source

### ONNX Runtime Version or Commit ID

a0db2187ee24115d75053ee4a87c5411191d591b

### ONNX Runtime API

Python

### Architecture

X64

### Execution Provider

Default CPU

**Comment by snnn** [comment]
See what change was added in #19745 for reshape. You need to do something similar for the new opset 22.
